includes/Checker/Check_Repository.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Check_Repository
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+/**
+ * Check Repository interface.
+ *
+ * @since n.e.x.t
+ */
+interface Check_Repository {
+
+	/**
+	 * Bitwise flag for static type checks.
+	 *
+	 * @since n.e.x.t
+	 * @var int
+	 */
+	const TYPE_STATIC = 1;
+
+	/**
+	 * Bitwise flag for runtime type checks.
+	 *
+	 * @since n.e.x.t
+	 * @var int
+	 */
+	const TYPE_RUNTIME = 2;
+
+	/**
+	 * Bitwise flag for all check types.
```

Comment:
Maybe add a comment to clarify why this value is `3`.
```suggestion
	 * Bitwise flag for all check types.
	 *
	 * This is the same as `TYPE_STATIC | TYPE_RUNTIME`.
```

Reply:
Updated